lib/wallaby/browser.ex

```diff
@@ -201,8 +201,10 @@ defmodule Wallaby.Browser do
   tests are run in.
 
   Pass `[{:name, "some_name"}]` to specify the file name. Defaults to a timestamp.
+  Pass `[{:log, true}]` to log the location of the screenshot to stdout. Defaults to false.
   """
   @type take_screenshot_opt :: {:name, String.t}
+  @type take_screenshot_opt :: {:log, boolean }
```

Comment:
I think the two definitions of `take_screenshot_opt` need to be combined like this.

```elixir
@type take_screenshot_opt :: {:name, String.t} | {:log, boolean}
```